File: EleuSrcCs/Ex1.cs
using Eleu;
using static Eleu.ValueStatics;

public class Ex1
{
	static Value a = CreateNumberVal(1);
	
	public static Value Main()
	{
		var b = CreateNumberVal(2);
		a = CreateStringVal(@"Hallo");
		{
			var c = CreateNumberVal(3);
		}
		Console.WriteLine(b);
		Console.WriteLine(a);
		
		return NilValue;
	}
}

File: EleuSrcNodejs/Ex1.js
var ελευ = require('./EleuCore');
var a = ελευ.CreateNumberVal(1);

{
	var b = ελευ.CreateNumberVal(2);
	a = ελευ.CreateStringVal("Hallo");
	{
		var c = ελευ.CreateNumberVal(3);
	}
	console.log((b).toString());
	console.log((a + b).toString());
}

